package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "fulminologist",
+  "version": "0.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "less": "PATH=./node_modules/.bin:$PATH lessc --autoprefix src/less/main.less livereveal/main.css",
```

Comment:
This need docs at the readme (well, we have to build a doc, but for the moment README is enough...